packages/core/src/types.tsx

```diff
@@ -171,7 +171,7 @@ export class PrivateValueStore<T extends [any, any, any]> {
   protected ''?: T;
 }
 
-type NavigationHelpersCommon<
+export type NavigationHelpersCommon<
```

Comment:
Exporting this makes it part of the public API which I'd avoid. This should never be used directly, the `NavigationHelpers` type should be used instead.